bin/release_automation.sh

```diff
@@ -34,19 +34,19 @@ if [[ ! "$CURRENT_BRANCH" =~ "^develop$|^main$|^release/.*" ]]; then
 fi
 
 # Confirm branch is clean
-[[ -z "$(git status --porcelain)" ]] || { git status; printf "\nUncommitted changes found. Aborting release script...\n"; exit 1; }
+[[ -z "$(git status --porcelain)" ]] || { git status; abort "Uncommitted changes found. Aborting release script..."; }
 
 # Ask for new version number
-CURRENT_VERSION_NUMBER=$(./node_modules/.bin/json -f package.json version)
+CURRENT_VERSION_NUMBER=$(jq '.version' package.json --raw-output)
```

Comment:
Might be worth adding an early check that jq is available so that if it's not available the script can abort with a clear error message.

Reply:
Makes sense 👍